course2/meeting4/CustomString.h

```diff
@@ -0,0 +1,195 @@
+/*
+ * CustomString.h
+ *
+ *  Created on: Jan 20, 2016
+ *      Author: matthijs
+ */
+
+#ifndef CUSTOMSTRING_H_
+#define CUSTOMSTRING_H_
+
+#include <string>
+#include <cstring>
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <memory>
+
+typedef std::unique_ptr<char[]> cstr_ptr;
+
+class CustomString
+{
+public:
+	CustomString() {}
+
+	CustomString(const char* str) {
+		copyCharArray(str);
+	}
+
+	int length() { return mChars.size(); }
```

Comment:
Whoops didn't mean to include this (and the files below) in the pull request. It's only about main.cc